akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -779,7 +779,7 @@ final class TlsModulePhase(settings: ActorMaterializerSettings, materializer: Ph
     publishers(out.id)
 
   def takePublisher(slot: Int, publisher: Publisher[Any]): Unit =
-    publisher.subscribe(FanIn.SubInput[Any](tlsActor, slot))
+    publisher.subscribe(FanIn.SubInput[Any](tlsActor, 1-slot))
```

Comment:
Is that an artifact of how shape ports are enumerated? Or why do these need to be swapped?